inference-engine/tests/functional/plugin/gna/shared_tests_instances/single_layer_tests/concat.cpp

```diff
@@ -22,9 +21,8 @@ std::vector<std::vector<std::vector<size_t>>> inShapes = {
 
 std::vector<InferenceEngine::Precision> netPrecisions = {InferenceEngine::Precision::FP32,
                                                          InferenceEngine::Precision::FP16};
-
 // TODO: Issue:  26421
```

Comment:
I am not sure if this issue can be closed since not all original test parameters are supported: (0, 2, 3 axes are not supported and 1-input concat produces an error while we support only IRV7 without Result layer and concat removal inside the plugin causes output layer change). I'd leave this comment here and update the ticket with actual problems.